app/controllers/catalog_controller.rb

```diff
@@ -526,6 +526,21 @@ def send_emails_to_all_recipients
     flash[:success] = I18n.t('blacklight.email.success')
   end
 
+  def augment_solr_document_json_response(documents)
+    documents.map do |document|
+      next document.as_json unless document&.access_panels&.online?
+
+      links = document&.access_panels&.online&.links&.map do |link|
+        html = link.stanford_only? ? "<span class=\"stanford-only\">#{link.html}</span>" : link.html
+
+        "<span class=\"online-label\">Online</span> #{html}"
```

Comment:
FWIW, I don't think we should be providing this label in the API.  We were already doing this in the Articles+ response and the bento app has a shared partial for both catalog and articles+ results.

This kept us consistent, but if this seems agreeable, I would like to ticket (and happily work on) adding PRs to SearchWorks and Bento to move these labels to the search service for each endpoint respectively (and we would then need to sync up on deploying those changes out).

Reply:
@jkeck This sounds like a good change! Feel free to ticket 🎟